PWGMM/Mult/Tasks/heavy-ion-mult.cxx

```diff
@@ -68,13 +71,18 @@ static constexpr TrackSelectionFlags::flagtype trackSelectionDCA =
 static constexpr TrackSelectionFlags::flagtype trackSelectionDCAXYonly =
   TrackSelectionFlags::kDCAxy;
 
+static constexpr int CentHistArray{15};
+std::shared_ptr<TH1> h1MultHistCentFT0C[CentHistArray];
```

Comment:
It is recommended to use `std::array` rather than C-arrays. In this case `std::array<std::shared_ptr<TH1>, CentHistArray> h1MultHistCentFT0C;`